src/libmongoc/tests/TestSuite.c

```diff
@@ -697,6 +734,8 @@ TestSuite_PrintHelp (TestSuite *suite) /* IN */
       "    -s, --silent          Suppress all output.\n"
       "    -F FILENAME           Write test results (JSON) to FILENAME.\n"
       "    -d                    Print debug output (useful if a test hangs).\n"
+      "    --skip-failing FILENAME      Skip known failing tests.\n"
```

Comment:
Recommend updating alignment of other option descriptions:
```c
      "Options:\n"
      "    -h, --help                Show this help menu.\n"
      "    --list-tests              Print list of available tests.\n"
      "    -f, --no-fork             Do not spawn a process per test (abort on "
      "first error).\n"
      "    -l, --match PATTERN       Run test by name, e.g. "
      "\"/Client/command\" or \"/Client/*\". May be repeated.\n"
      "    --ctest-run TEST          Run only the named TEST for CTest\n"
      "                              integration.\n"
      "    -s, --silent              Suppress all output.\n"
      "    -F FILENAME               Write test results (JSON) to FILENAME.\n"
      "    -d                        Print debug output (useful if a test "
      "hangs).\n"
      "    --skip-failing FILENAME   Skip known failing tests.\n"
      "    --skip-flaky FILENAME     Skip known flaky tests.\n"
      "    -t, --trace               Enable mongoc tracing (useful to debug "
      "tests).\n"
```

Reply:
Coming in a new commit shortly.